### ISSUE TITLE
Unable to redeploy auth_init or auth_callback - even after making no changes at all

### ISSUE THREAD

**Issue by hughsheehy** [googlecodelabs/gcf-gmail-codelab#14]
Another problem on this lab - probably related to the Node 8 change. 

I can deploy the auth_init and auth_callback functions, despite getting a warning about Node 8 being deprecated. 

Then, I was getting frustrated because I'd get a failure to deploy when trying to add the callback URL (as per the lab instructions, page 4). So I tried to redeploy with no changes at all. And that failed. 

Lots of errors apparently related to secrets and needing GCP_PROJECT set.   Since most back end I do is in GO, this is not stuff I know.

**Comment by vmanpro** [comment]
+1 here
are you able to solve the problem?

**Comment by hughsheehy** [comment]
> +1 here
> are you able to solve the problem?

No. I had other things I needed to do, so stepped away. Would love to go back to it, but not if I have to figure out the steps across several versions of Node.

**Comment by kbroughton** [comment]
i found the dev loop really clunky and error prone. I just `gcloud functions delete` and re-deply for more reliable results.

**Comment by amurciasu** [comment]
you can try this https://github.com/googlecodelabs/gcf-gmail-codelab/issues/18#issuecomment-1039142000
